storage/client/queue/src/main/java/com/azure/storage/queue/models/DequeuedMessage.java

```diff
@@ -1,6 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
Do the other AutoRest generated classes that had manual changes have this?

Reply:
This class has been deleted from my local. No idea why it shows up. Do we need this anymore?
I will double check manual changed model for the license header. Do we want to have the third line or remove the one?

Reply:
Discussed offline. Keep the five classes under model using the manual license header.
UpdatedMessage
DequeuedMessage
QueueProperties
PeekedMessage
QueuesSegmentOptions